src/features/auth/register-form/use-register-form-state.ts

```diff
@@ -1,42 +1,36 @@
 import { useState, ChangeEvent } from "react";
 
 export default function useRegisterFormState() {
-  const [name, setName] = useState<string>("");
-  const [lastname, setLastname] = useState<string>("");
-  const [email, setEmail] = useState<string>("");
-  const [password, setPassword] = useState<string>("");
-  const [confirmPassword, setConfirmPassword] = useState<string>("");
+    const [name, setName] = useState<string>("");
+    const [email, setEmail] = useState<string>("");
+    const [password, setPassword] = useState<string>("");
+    const [confirmPassword, setConfirmPassword] = useState<string>("");
```

Comment:
Why not null?
```suggestion
    const [name, setName] = useState<string|null>(null);
    const [email, setEmail] = useState<string|null>(null);
    const [password, setPassword] = useState<string|null>(null);
    const [confirmPassword, setConfirmPassword] = useState<string|null>(null);
```
improve with
```
type InputText = string | null;
```
```suggestion
    const [name, setName] = useState<InputText>(null);
    const [email, setEmail] = useState<InputText>(null);
    const [password, setPassword] = useState<InputText>(null);
    const [confirmPassword, setConfirmPassword] = useState<InputText>(null);
```

Reply:
Eso me da este warning 
![image](https://github.com/EquestriaReapers/ER-frontend/assets/70321814/8d359e0d-0b88-404a-ac46-d338859f74c6)